capture/src/capture.cpp

```diff
@@ -178,18 +185,27 @@ int main( int argc, char** argv )
     }
     const auto t1 = std::chrono::high_resolution_clock::now();
 
+    const char *ansiReset = isTTY ? "\033[0m" : "";
+    const char *ansiBold = isTTY ? "\033[1m" : "";
+    const char *ansiBlack = isTTY ? "\033[30m" : "";
+    const char *ansiRed = isTTY ? "\033[31m" : "";
+    const char *ansiGreen = isTTY ? "\033[32m" : "";
+    const char *ansiYellow = isTTY ? "\033[33m" : "";
+    const char *ansiMagenta = isTTY ? "\033[35m" : "";
+    const char *ansiCyan = isTTY ? "\033[36m" : "";
```

Comment:
Please move this code earlier (e.g. next to the `isTTY` initialization). Then please use these variables in the `if (isTTY) {` block, to improve readability. Maybe an alternate way of encoding colors would be better here? Something like:

```c++
#define AnsiRed "\033[31m"
...
const char *ansiRed = isTTY ? AnsiRed : "";
...
printf( AnsiRed "error" ... );
```

Of course the `%s` indirection would need to stay below, as it is dynamic, but in the code above the check is at entry into the block, so colors are always used there.

Reply:
done... went a bit farther, introduced `AnsiPrintf` function.

Reply:
It looks fine, but please adapt this function to the coding style.